tools/sof-kernel-log-check.sh

```diff
@@ -204,8 +204,8 @@ ignore_str="$ignore_str"'|usb 2-.: .'
 
 # TGL devices with USB 3.1 devices, issues reported by sof-test
 # BugLink: https://github.com/thesofproject/sof-test/issues/482
-ignore_str="$ignore_str"'|usb 3-.: device descriptor read/64, error .'
-ignore_str="$ignore_str"'|usb 3-.: device not accepting address ., error .'
+ignore_str="$ignore_str"'|usb 3-..: device descriptor read/64, error'
```

Comment:
I don't think this is going to work well,  the `usb 3-..` is not going to work if we have `usb 3-4`, and we have!!!
This is better:
```
ignore_str="$ignore_str"'|usb 3-.+: device descriptor read/64, error .+'
ignore_str="$ignore_str"'|usb 3-.+: device not accepting address .+, error .+'
```

there is one more usb error in report 1021
```
[ 6688.142149] kernel: usb usb3-port10: unable to enumerate USB device
```
add an re like this
```
usb usb.-port.+: unable to enumerate USB device
```

Reply:
all above re are tested locally.